apps/spark/build.gradle

```diff
@@ -68,8 +68,13 @@ dependencies {
     implementation 'com.fasterxml.woodstox:woodstox-core:6.2.7'
 
     // open telemetry related classed. Latest Okhttp version is 4.10.0, pinning to 4.9.3 to avoid dependency issues
-    implementation 'com.squareup.okhttp3:okhttp:4.9.3'
+    implementation 'com.squareup.okhttp3:okhttp:' + ok_http3_version
+    implementation 'com.squareup.okhttp:okhttp:2.7.5'
     implementation 'com.squareup.okio:okio:3.2.0'
+    implementation 'com.squareup.okio:okio-jvm:3.2.0'
+    implementation 'org.jetbrains.kotlin:kotlin-stdlib:2.0.20'
```

Comment:
Is there a specific dependency on kotlin lib?

Reply:
it's pulled com.square.up, I'm pinning it here to a specific version